test/regression/2016-04-14-sdiv-2.c_

```diff
@@ -0,0 +1,25 @@
+// XFAIL: *
```

Comment:
OK, I thought it's intentional.  But is it possible to add tests marked as disabled for lit?